L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -510,6 +510,18 @@
                            )
 
 ## GEM-CSC ILT in ME1/1
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+run2_GEM_2017.toModify( cscTriggerPrimitiveDigis,
+                        GEMPadDigiProducer = cms.InputTag("simMuonGEMPadDigis"),
```

Comment:
run2_GEM_2017 is a part of Run2_2017 era and will be (it is, already) used for data taking.
Clearly, simDigis do not exist in this case and normal unpacked digis should be used.
If you need MC-only setup, we should make something like run2_GEM_2017_MCTest modifier and define a Run2_2017_GEMMCTest era if it's needed for private tests.